backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/vdscommands/VmBackupVDSParameters.java

```diff
@@ -10,20 +11,23 @@
 
     private VmBackup vmBackup;
     private boolean requireConsistency;
-    // Map between the backed-up disk ID to the created scratch disk image path.
-    private Map<Guid, String> scratchDisksMap;
+    // Map between the backed-up disk ID to the created scratch disk image
+    // and the path to it after the scratch disk was prepared.
+    private Map<Guid, ScratchDiskInfo> scratchDisksMap;
 
     public VmBackupVDSParameters() {
     }
 
     public VmBackupVDSParameters(Guid vdsId, VmBackup vmBackup) {
-        this(vdsId, vmBackup, false);
+        this(vdsId, vmBackup, false, new HashMap<>());
     }
 
-    public VmBackupVDSParameters(Guid vdsId, VmBackup vmBackup, boolean requireConsistency) {
+    public VmBackupVDSParameters(Guid vdsId, VmBackup vmBackup, boolean requireConsistency,
+                                 Map<Guid, ScratchDiskInfo> scratchDisksMap) {
```

Comment:
also here - formatting

Reply:
Again, my IDE formats this way by default.
Although I did reformat to make each of the 4 method parameters on a separate line vs. 3 on one line and 1 on the other,

Reply:
Done.
Changed the indentation, each method parameter on a separate line.